pkg/yurthub/cachemanager/storage_wrapper.go

```diff
@@ -131,41 +118,29 @@ func (sw *storageWrapper) Get(key string) (runtime.Object, error) {
 	} else {
 		UnstructuredObj = new(unstructured.Unstructured)
 	}
-	obj, gvk, err := sw.backendSerializer.Decode(b, nil, UnstructuredObj)
+	obj, gvk, err := sw.universalDecoder.Decode(b, nil, UnstructuredObj)
```

Comment:
why not use `jsonEncoder` here?

Reply:
I encountered the problem that the response from APIServer is not JSON format, but protobuf format. So I replace the decoder as universalDecoder to solve it. Since then, a lot of changes have be placed on code. Maybe we do not need to use universalDecoder here anymore.

Reply:
fixed